pkg/manifest/properties.go

```diff
@@ -102,11 +102,18 @@ func (p Properties) Layout() EPUBLayout {
 }
 
 func (p Properties) Encryption() *Encryption {
-	mp, ok := p.Get("encrypted").(map[string]interface{})
+	v := p.Get("encrypted")
+	if v == nil {
+		return nil
+	}
+	if enc, ok := v.(Encryption); ok {
+		// Is already an Encryption struct
```

Comment:
Link `Properties` must only contain JSON data specifically to avoid this ambiguity and facilitate the serialization of the manifest to JSON. If there are `Encryption` or `HashList` objects in `Properties`, they need to be serialized at the location where they are added to `Properties`.

Reply:
@mickael-menu I changed it, look OK?